src/styles/blocks/card.scss

```
@@ -0,0 +1,130 @@
+.card {
+  height: 520px;
+
+  &__content {
+    max-width: 370px;
+    margin-top: 50px;
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+  }
+
+  &__image {
+    width: 670px;
+    height: 520px;
+  }
+
+  &__data {
+    display: flex;
+    margin-bottom: 30px;
+  }
+
+  &__exposition {
+    margin-right: 26px;
+    text-transform: uppercase;
+    font-family: $text-font;
+    font-weight: 500;
+    font-size: 14px;
+    color: $grey-color;
+  }
+
+  &__time {
+    font-family: $text-font;
+    font-weight: bold;
+    font-size: 14px;
+    line-height: 150%;
+    color: $green-color;
+  }
+
+  &__title {
+    margin: 0 0 15px;
+    font-family: $title-font;
+    font-size: 36px;
+    line-height: 100%;
+    color: $title-color;
+
+    &::after {
+      content: "";
+      display: inline-block;
+      width: 15px;
+      height: 15px;
+      margin: 0 0 2px 24px;
+      background-color: $orange-color;
+    }
+  }
+
+  &__text {
+    font-family: $text-font;
+    font-weight: 300;
+    font-size: 16px;
+    line-height: 150%;
+    color: $text-color;
+  }
+
+  @media (min-width: $large-screen) {
+    &__image {
+      width: auto;
+    }
+
+    &__content {
+      margin: 0 130px;
+    }
+  }
+
+  @media (max-width: 1279px) {
+    & {
+      height: 720px;
+    }
+
+    &__content {
+      margin-top: 30px;
+    }
+  }
```

```suggestion
  @media (max-width: 1279px) {
      height: 720px;

    &__content {
      margin-top: 30px;
    }
  }
```
'
это лишнее давай уберем везде